sdks/universal-router-sdk/src/utils/constants.ts

```diff
@@ -1,8 +1,8 @@
 import { BigNumber } from 'ethers'
 
 type ChainConfig = {
-  v3Router: string
-  v4Router: string
+  v1_2Router: string
```

Comment:
we'll probably eventually have to do something similar with creationBlock. prefer alpha order based on thing, not version?? hence `routerV1_2`, `routerV2_0`? then we can have creationBlockV-whatever down the line

Reply:
great callout!